extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchMergeAggregatorFactory.java

```diff
@@ -142,6 +151,22 @@ public VectorAggregator factorizeVector(VectorColumnSelectorFactory selectorFact
     );
   }
 
+  private void validateInputs(@Nullable ColumnCapabilities capabilities)
```

Comment:
Please add a Javadoc here as to what this method does, and what those validations are and why.

Reply:
I'll add a basic javadoc, not sure if it'll be useful though. I lack detailed context of rationale behind each unsupported type for each aggregator factory.

Reply:
We should mention the types that are supported.

Reply:
@LakshSingla I'm confused on this.

For example, `HllSketchApproxCountDistinctSqlAggregator` allows String and Numeric column types, but `HllSketchMergeAggregatorFactory#validateInputs` doesn't?